src/sentry/static/sentry/app/components/commitAuthorStats.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+
+import LoadingIndicator from '../components/loadingIndicator';
+import LoadingError from '../components/loadingError';
+import Avatar from '../components/avatar';
+
+import TooltipMixin from '../mixins/tooltip';
+import ApiMixin from '../mixins/apiMixin';
+
+const CommitBar = React.createClass({
+  propTypes: {
+    whole: React.PropTypes.number.isRequired,
```

Comment:
I believe 'whole' refers to the total commit number to specify the size of the commitBar, and 'part' is the number of commits for the specific contributor. Do you prefer I change these names to clarify?